py/server/deephaven/_jcompat.py

```diff
@@ -1,95 +0,0 @@
-#
-#   Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
-#
-""" This module provides Java compatibility support including convenience functions to create some widely used Java
```

Comment:
I assume this was unused?

Reply:
yes, we decided to make it public but the renaming change and the py projects reorg work somehow left both in